### PR TITLE
OSDOCS-2185: GCE Auto Migration (TP)

### PR DESCRIPTION
**Pull request by lpettyjo**
4.9+

[OSDOCS-2185:](https://issues.redhat.com/browse/OSDOCS-2185) GCE added to auto migration feature (Tech Preview).

**Preview**: https://deploy-preview-34443--osdocs.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi-migration?utm_source=github&utm_campaign=bot_dp

PTAL @bertinatto, @chao007

### CODE REVIEW

File: modules/persistent-storage-csi-migration-overview.adoc [hunk 1]
```diff
@@ -13,6 +13,8 @@ The following drivers are supported:
 
 * OpenStack Cinder
 
+* Google Compute Engine Persistent Disk (GCE PD) (in-tree) and Google Cloud Platform (GCP) PD (CSI)
```
Comment (kalexand-rh) [inline modules/persistent-storage-csi-migration-overview.adoc:16]: I suggest splitting these items into separate lines and/or considering either not using the product abbreviations or styling the types in something other than parentheses. (This line's hard to follow as-is.)

File: modules/persistent-storage-csi-migration-enable.adoc [hunk 2]
```diff
@@ -33,12 +33,14 @@ spec:
 ----
 <1> Enables automatic migration for AWS EBS and Cinder.
 +
-You can specify CSI automatic migration for a selected CSI driver by setting `CustomNoUpgrade` `featureSet` and for `featuregates` either:
+You can specify CSI automatic migration for a selected CSI driver by setting `CustomNoUpgrade` `featureSet` and for `featuregates` to one of the followoing:
```
Comment (bertinatto) [inline modules/persistent-storage-csi-migration-enable.adoc:36]: typo: following

**Review by bertinatto — COMMENTED** [review]
/lgtm
from engineering PoV. There's a minor typo that you may want to fix before merging, though.

**Review by bertinatto — COMMENTED** [review]
/lgtm
